docs/features/duplicate_granule_handling.md

```diff
@@ -0,0 +1,67 @@
+---
+id: duplicate_granule_handling
+title: Cumulus Duplicate Granule Handling
+hide_title: false
+---
+
+Collections and Workflows can be configured to assign a unique `granuleId` to each Granule. This is useful in cases where two Granules share the same `granuleId` but must both be ingested into the Cumulus system. Previously, a Granule with a conflict on the `granuleId` key would cause an ingest failure for that Granule.
+
+## Granule Uniquification
+
+Cumulus has implemented a set of changes that allow multiple Granules with identical `granuleIds` to be ingested and managed independently. This is done by replacing the `granuleId` with a unique value and storing the original `granuleId` value as a new field, `producerGranuleId`. This new `producerGranuleId` field can then be used to track and correlate the Granule to the original data holdings.
+
+### Granule Uniquification Algorithm and Task Component
+
+The process Cumulus Core uses to generate a unique `granuleId` can be [found here](https://github.com/nasa/cumulus/blob/feature/duplicate-granule-924/packages/ingest/src/granule.ts#L479-L487).
```

Comment:
The link will change, maybe just link to future master branch and use the package and method name.

Reply:
Ah yes. I forgot to call this out. I updated the links to use master and I'll find a ticket to add an AC that double-checks these links at the end once the feature is merged.

Reply:
https://github.com/nasa/cumulus/pull/4021/commits/5d24744de181c7ebbf52becf19919ba948cd5533